cmd/fyne_demo/tutorials/dialog.go

```diff
@@ -143,6 +144,34 @@ func dialogScreen(win fyne.Window) fyne.CanvasObject {
 				log.Println("Please Authenticate", username.Text, password.Text)
 			}, win)
 		}),
+		widget.NewButton("Form Dialog (Login Form)", func() {
+			userRx := validation.NewRegexp(`^[A-Za-z0-9_-]+$`, "username can only contain letters, numbers, '_', and '-'")
+			username := widget.NewEntry()
+			username.Validator = userRx
+			passRx := validation.NewRegexp(`^[A-Za-z0-9_-]+$`, "password can only contain letters, numbers, '_', and '-'")
```

Comment:
Very minor comment. You could just set this directly instead of creating a new variable.